kutt.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/bugsnag/microkit/clog"
+	"github.com/pkg/errors"
+	"github.com/spf13/viper"
+)
+
+type Client struct {
+	HTTPClient *http.Client
+	ApiKey     string
+	BaseURL    string
+	UserAgent  string
+}
+
+type SubmitParams struct {
+	URL      string `json:"target"`
+	ExpireIn string `json:"expire_in"`
+}
+
+func NewClient() *Client {
```

Comment:
Should probably be named:

```suggestion
func NewLinksClient() *Client {
```